htdocs/document.php

```diff
@@ -168,6 +168,7 @@ function llxFooter()
 			$conf->entity = $entity;
 			$conf->setValues($db);
 		}
+		$fullpath_original_file = DOL_DATA_ROOT . '/' . $ecmfile->filepath . '/' . $ecmfile->filename;
```

Comment:
The original_path is an information on original filename.
Not used for the moment, but must not be forged to contains another value than the one provided into the form by end user.